src/CollectionsFrameWorkChallenge/ArrayListChallenge.java

```diff
@@ -0,0 +1,47 @@
+package CollectionsFrameWorkChallenge;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+public class ArrayListChallenge {
+    public static void main(String[] args) {
+        List<String> colorList = new ArrayList<>();
+        colorList.add("Red");
+        colorList.add("Blue");
+        colorList.add("Green");
+        colorList.add("Black");
+        System.out.println(colorList);
+        for (String element : colorList) {
```

Comment:
when you have only one class with lots of lines of code, you can try to refactor that method and create smaller oners and just make use of them in main()
for example, you can have a method called `printStringList(List<String> list)` and inside you can have that for loop


for (String element : colorList) {
            System.out.println(element);
        }